### PR TITLE
(chore) Set up coveralls

### PR DESCRIPTION
**Pull request by mtlewis**

### CODE REVIEW

**Comment by mtlewis** [comment]
This will start passing once coveralls has been authorised in the schibsted org.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # jsx-pdf
 
-[![Build Status](https://travis-ci.org/schibsted/jsx-pdf.svg?branch=master)](https://travis-ci.org/schibsted/jsx-pdf)
+[![Build Status](https://travis-ci.org/schibsted/jsx-pdf.svg?branch=master)](https://travis-ci.org/schibsted/jsx-pdf) [![Coverage Status](https://coveralls.io/repos/github/mtlewis/jsx-pdf/badge.svg?branch=master)](https://coveralls.io/github/mtlewis/jsx-pdf?branch=master)
```
Comment (yaroslav-borcheninov) [inline README.md]: Should it be `schibsted/jsx-pdf`?

Comment (mtlewis) [reply README.md]: Yep, thanks :)

Comment (mtlewis) [reply README.md]: Fixed

File: package.json [hunk 2]
```diff
@@ -16,7 +16,9 @@
     "dev": "yarn build -- --watch src",
     "build": "babel src --out-dir dst --ignore '**/*.test.js'",
     "lint": "eslint \"{src,example}\"",
-    "test": "yarn lint && jest",
+    "test": "yarn lint && yarn test:unit${TRAVIS:+':coveralls'}",
+    "test:unit": "jest --coverage",
+    "test:unit:coveralls": "jest --coverage --coverageReporters=text-lcov | coveralls",
```
Comment (yaroslav-borcheninov) [inline package.json]: Just curious. I can't find `coverageReporters` option in the jest docs - is this something they support unofficially?

Comment (mtlewis) [reply package.json]: Good question - this is taken from the [node-coveralls docs](https://github.com/nickmerwin/node-coveralls#jest).  I'll see if I can dig up some docs.

Comment (mtlewis) [reply package.json]: From the top paragraph of the [Jest CLI docs](https://jestjs.io/docs/en/cli):

> Every one of Jest's Configuration options can also be specified through the CLI.
 
So it looks like the docs for the CLI only cover CLI-specific things, but everything else from config (including [coverageReporters](https://jestjs.io/docs/en/configuration#coveragereporters-array-string)) is also supported via CLI.

File: package.json [hunk 3]
```diff
@@ -16,7 +16,9 @@
     "dev": "yarn build -- --watch src",
     "build": "babel src --out-dir dst --ignore '**/*.test.js'",
     "lint": "eslint \"{src,example}\"",
-    "test": "yarn lint && jest",
+    "test": "yarn lint && yarn test:unit${TRAVIS:+':coveralls'}",
```
Comment (yaroslav-borcheninov) [inline package.json]: nit: Echoing this script gives me `yarn lint && yarn test:unit':coveralls'`. Does it work without quotes?

Comment (mtlewis) [reply package.json]: Yeah - actually the single quotes are a typo, and it should be either unquoted or double-quotes.  I think maybe double quotes makes the whole thing a bit more readable, so I'll change it to that :)

Comment (mtlewis) [reply package.json]: Oh... but then I've got to escape them which makes the whole thing look messy.  OK - unquoted it is :)

**Pull request merged**